### PR TITLE
Angular 0.2.56: Editor, Media Dialog, Froala, & Froala Paste Functionality

### PR DESCRIPTION
**Pull request by alexgurrola**
Adds:

- Editor: Handling for img tags with base64 data
- Froala: Paste Uploads Images
- Froala: Font & Style Config
- Froala: Remote Image Uploads
- Froala: Tag Filters
- Media-Dialog: Snackbar Messages
- Media-Dialog: Embedly Support
- Media-Dialog: Direct Link Support
- Media-Dialog: Video Support (URL & Embed)
- Media-Dialog: Audio Support
- Media-Dialog: PDF Support
- Media-Dialog: Doc Support
- Media-Dialog: Vimeo Thumbnail Support
- Media-Dialog: Youtube Thumbnail Support
- Media-Dialog: Video Text Fallback

Fixes:

- Froala: Catastrophic Pastes (Inserting Images created base64 data and overloaded the system)
- Froala: Z-Index (Quick Insert, Table, & Media Edit Dialogs previously appeared behind the editor)

Signed-off-by: alexgurrola <alexgurrola1@gmail.com>

### CODE REVIEW

File: packages/angular/src/editor/editor.component.ts [hunk 1]
```diff
@@ -370,6 +370,7 @@ export class EditorComponent extends RootComponent implements OnInit, TriggerInt
     froalaConfig: LooseObject = {
         attribution: false,
         key: Stratus.Environment.get('froalaKey'),
+        zIndex: 9999,
```
Comment (apocist) [inline packages/angular/src/editor/editor.component.ts:373]: I'm assuming you don't expect any popups to appear over the editor

**Review by apocist — APPROVED** [review]
I just worry about my comment on the zIndex, as angularjs has a max zIndex of 100. Not sure how the zindexing works here, but I'm wondering if it'll hover above popups/dialogs

File: packages/angular/src/editor/editor.component.ts (hunk 1 above)
Comment (alexgurrola) [reply packages/angular/src/editor/editor.component.ts:373]: Presumably, the `zIndex` field is only used for their tooltips.  I have an Angular+ Dialog for the Media Selection that goes over the entire editor without any problems.

**Comment by alexgurrola** [comment]
> I just worry about my comment on the zIndex, as angularjs has a max zIndex of 100. Not sure how the zindexing works here, but I'm wondering if it'll hover above popups/dialogs

The editor doesn't appear to have any conflict with the `zIndex` for the AngularJS or the Angular+ Media Selector dialogs.

**Pull request merged**